sdk/src/main/res/layout/hcaptcha_fragment.xml

```diff
@@ -22,7 +22,7 @@
             android:layout_width="150dp"
             android:layout_height="wrap_content"
             android:adjustViewBounds="true"
-            android:src="@drawable/ic_logo"
+            android:src="@drawable/ic_hcaptcha_logo"
```

Comment:
Please do a version bump and add a changelog.

Reply:
Done